### ISSUE TITLE
Rule `no-incorrect-submodule`

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/eslint-plugin-lodash-fp#1]
This rule should throw an error when importing a named export or destructuring a required module, when the module is a Lodash method path

### Fail
```js
import {find} from 'lodash/find';
import {find as f} from 'lodash/find';
import {find} from 'lodash/fp/find';
var {find} = require('lodash/find');
var {find: f} = require('lodash/find');
var {find} = require('lodash/fp/find');
```

### Pass

```js
import {find} from 'lodash';
import {find as f} from 'lodash';
import {find} from 'lodash/fp';
import find from 'lodash/find';
import find from 'lodash/fp/find';
var {find} = require('lodash');
var {find: f} = require('lodash');
var {find} = require('lodash/fp');
var find = require('lodash/find');
var find = require('lodash/fp/find');
```

(if someone has a better name for it, let me know)
